tests/copy-paste.js

```diff
@@ -76,7 +76,6 @@ define([
                         "labelItext:hin-default",
                         "constraintMsgItext:en-default",
                         "constraintMsgItext:hin-default",
-                        "constraintMsgItext",
```

Comment:
Itext's shouldn't be copied because they're autoid. I don't think constraintMsgAttr needs to be copied either, but it's a bigger lift to do that.